### PR TITLE
add: LFX government proposal

### PR DESCRIPTION
**Pull request by IcyFeather233**
**What type of PR is this?**

/kind documentation

the proposal of [CNCF - KubeEdge: Domain-specific large model benchmarks: the edge perspective (2025 Term 1)](https://mentorship.lfx.linuxfoundation.org/project/e3fc44d9-9ddd-42e6-a9be-8f6c2a114672)

### CODE REVIEW

**Review by MooreZheng — CHANGES_REQUESTED** [review]
Overall the RAG idea is impressive. It would be important for not only gov related LLM but also all domain-specific LLMs. 

There are some issues yet to discuss. 
1.  A stucture design is needed. Integration of RAG, e.g., API/ wheel etc, requires careful design.
2. RAG database and data format need preparation and further research. 
3. The test setting of using which part of the whole dataset, such as the RAG dataset, the fine-tune dataset, the few-shot prompt dataset, the known test set, and the unknown test set for the same or different nodes, is undoubtedly complicated. Besides, there is also the algorithm selection issue for RAG and LLM,  together with the above dataset issue, leading to a more complicated test setting.

**Review by MooreZheng — COMMENTED** [review]
Marked: This proposal PR is related to https://github.com/kubeedge/ianvs/issues/177

**Review by MooreZheng — CHANGES_REQUESTED** [review]
The new version integrates different RAG algorithms with the introduced feature discussed. Good to see that. 
 
The previous [comments](https://github.com/kubeedge/ianvs/pull/186#pullrequestreview-2681863608) are yet to discussed. Among all concerns, the test setting is especially important, which would affect the structure design. How many datasets, algorithms and hyper-parameters would be introduced and integrated? Will the integrations affect the ianvs core, e.g., test environment, and why? 

(BTW, the proposal PR seems to include non-necessary documents from other commits.  Besides, proposal PR shall not include code. Thus cleaning is needed.)

**Comment by AryanNanda17** [comment]
Hello @IcyFeather233 
I noticed that some of the code changes in this PR are from the previous commits I did. It looks like your branch doesn’t include those commits, which might have caused the duplication. Could you check and rebase your branch on the latest main/develop branch?

**Review by MooreZheng — CHANGES_REQUESTED** [review]
The proposal is further improved and that is well done. 

As discussed on the routine meeting, there are suggestions for the current version
1. The overall architecture design is added and looks pretty good. It would be appreciated if the proposed dataset, metrics, and schemes are further highlighted. 
2. The test setting is added which is cool. Moreover, we might try to add test data setting. That is, for exact-match knowledge base, complete knowledge base, and excluding knowledge base, the average accuracy could be taken for test samples in order to evaluate the learning ability, searching ability, and generalization ability for the tested models 
3. Add limitations and future work session to state that the current proposal for domain-specific large model mainly focuses on introducing RAG but has not yet covered the testing of incremental training, fine tuning, and few-shot learning.

**Review by MooreZheng — CHANGES_REQUESTED** [review]
In this version, the setting and architecture are clear. Overall it looks fine and we are close to the final version. 

It would make the proposal better to consider the evaluation dimension, which shows the characteristics of a domain-specific model. That is, though showing arbitrary edge nodes (e.g., Beijing, Shanghai, etc) is clear, it might not always make sense to everyone (especially those who live far from the edge nodes ;D). 

One way to improve is to abstract the ability from measurement under different settings. For exact-match knowledge base, complete knowledge base, and excluding knowledge base, the average accuracy could be taken for test samples to evaluate the learning ability, searching ability, and generalization ability for the tested models.

**Review by hsj576 — APPROVED** [review]
/lgtm

**Comment by MooreZheng** [comment]
Related issue and PR:  
Phase 1 Gov LLM https://github.com/kubeedge/ianvs/issues/95 https://github.com/kubeedge/ianvs/pull/113 https://github.com/kubeedge/ianvs/pull/144
Phase 2 Gov RAG https://github.com/kubeedge/ianvs/issues/177 https://github.com/kubeedge/ianvs/pull/186 https://github.com/kubeedge/ianvs/pull/196
